### ISSUE TITLE
expected at least one HTML element tag inside a component

### ISSUE THREAD

**Issue by garthk** [phoenixframework/phoenix_live_view#1227]
### Environment

* Elixir version: 1.11.2 on OTP23
* Phoenix version: 1.5.6
* Phoenix LiveView version: 0.1.5.0-dev from git master at b16e4e3 and 0038bb1
* NodeJS version: v12.19.0
* NPM version: 6.14.8
* Operating system: macOS 10.15.7
* Browsers you attempted to reproduce this bug on (the more the merrier): Safari 14.0.1; Chrome 86.0.4240.198
* Does the problem persist after removing "assets/node_modules" and trying again? Yes

### Actual behavior

While diagnosing a connection issue, I've noticed we're getting this pretty consistently on page loads:

```plain
phx-FkgeUjV_cqRtXj_B mount:
 – (data)
expected at least one HTML element tag inside a component, but the component is empty:
 – ""
```

That'd be from [recursiveCIDToString/3](https://github.com/phoenixframework/phoenix_live_view/blob/b16e4e3982387fd6a5be03fddfd6cb17165a5ded/assets/js/phoenix_live_view.js#L612-L615), specifically:

```javascript
    if(!hasChildNodes && !hasChildComponents){
      logError(`expected at least one HTML element tag inside a component, but the component is empty:\n`,
               template.innerHTML.trim())
      return this.createSpan("", cid).outerHTML
```

It's nice to have the `innerHTML` as proof, I guess, but without any other detail it's hard to figure out which components or cids might be involved.

### Expected behavior

- No error would be nice
- If that error is our fault, it'd be super handy if there were some obvious clue as to how we might fix it

**Comment by garthk** [comment]
Pretty sure it's not #1035, as that was fixed in 64e75b2, but that's the only other search result for this error message. They didn't find it that helpful, either.

**Comment by chrismccord** [comment]
We need more information to say what's going one, but the only information the client has is the component ID and the HTML contents. Ensure your components are rendering at least one HTML tag, ie no text-only content or no `render` clause that returns nothing. If you think the javascript error could be rewritten to be more helpful, I would gladly accept a PR. Thanks!
